docs/source/kit/index.rst

```diff
@@ -0,0 +1,47 @@
+Kit
+===
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Quick Links:
+
+   arduino
+   battery
+   motor-board
+   power-board
+   servo-board
+   pi
+
+Overview
+------
+
+Besides the Raspberry Pi, there are 4 main controller boards to
+interface with your kit.
+
+================================== ================================== ================================== ========================== ==========================
+`Motor Board <motor-board.html>`__ `Power Board <power-board.html>`__ `Servo Board <servo-board.html>`__ `Arduino <arduino.html>`__ `Raspberry Pi <pi.html>`__
+================================== ================================== ================================== ========================== ==========================
+|Motor Board|                      |Power Board|                      |Servo Board|                      |Arduino|                  |Pi|
+================================== ================================== ================================== ========================== ==========================
+
+Other Parts
+-----------
+
+In addition to the boards, your kit will also contain:
+
+=============================================================== ======== =============
+Part                                                            Quantity Specification
+=============================================================== ======== =============
+`USB Hub <https://www.amazon.co.uk/-/dp/B01DYD3Q28/>`__         1
+`Micro USB Cable <https://www.amazon.co.uk/-/dp/B01EK87T9M/>`__ 2
+`USB B Cable <https://www.amazon.co.uk/-/dp/B00NH11KIK>`__      1
+`Large CamCon <http://uk.farnell.com/-/-/-/dp/3882275>`__       2        7.5mm
+`Medium CamCon <http://uk.farnell.com/-/-/-/dp/3881854>`__      2        5mm
+`Small CamCon <http://uk.farnell.com/-/-/-/dp/1717047>`__       3        3.81mm
+=============================================================== ======== =============
+ 
+.. |Motor Board| image:: /_static/kit/mcv4b.png
+.. |Power Board| image:: /_static/kit/pbv4.png
+.. |Servo Board| image:: /_static/kit/sbv4.png
+.. |Arduino| image:: /_static/kit/arduino.png
```

Comment:
`arduino.png` is a picture of legacy servo assembly. I suggest deleting it to prevent future confusion.., and renaming `arduino_headers.png` to `arduino.png`.

Reply:
To do in #18